jdaviz/core/astrowidgets_api.py

```diff
@@ -99,14 +99,7 @@ def center_on(self, point):
         else:  # pragma: no cover
             pix = point
 
-        width = self.state.x_max - self.state.x_min
-        height = self.state.y_max - self.state.y_min
-
-        with delay_callback(self.state, 'x_min', 'x_max', 'y_min', 'y_max'):
-            self.state.x_min = pix[0] - (width * 0.5)
-            self.state.y_min = pix[1] - (height * 0.5)
-            self.state.x_max = self.state.x_min + width
-            self.state.y_max = self.state.y_min + height

```

Comment:
I ended up rewriting this logic since I forgot it already existed... I'll have to think which way is more reliable or if there is any actual difference 🤔